### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GeraldC13**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #...
| ❓ Documentation  | yes

## Description

slightly updating project description and opening 1:1 tech discussions slots to onboard new users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,8 +63,10 @@ Build with ❤️️ from 🇫🇷 🇯🇵 🇬🇷 🇸🇪 🇺🇸
 
 ---
 
-## Collecting users feedback
+## Helping you get started
+
+We have 30 min slots for 1:1 [tech discussions and overviews](https://calendly.com/stephanejourdan/30min) twice a day to help you get started and collect users feedback.
 
-Have you launched driftctl ? Please take a moment to answer the [following questions](https://driftctl.typeform.com/to/fYUOOMj3) and help us improve the tool. 
+:warning: ** No sales involved ** You will deal with a technical team member of the project.
```
Comment (sundowndev) [inline README.md:70]: `**No sales involved**:`

**Review by sundowndev — CHANGES_REQUESTED** [review]
Just a small comment about formatting

File: README.md (hunk 1 above)
Comment (GeraldC13) [reply README.md:70]: good for me :)

File: README.md [hunk 2]
```diff
@@ -28,13 +28,11 @@
...
 ## Why driftctl ?
 
-Infrastructure as code is awesome, but there are too many moving parts: codebase, state file, actual cloud state. Things tend to drift.
+Infrastructure drift is a blind spot and a source of potential security issues.
+Drift can have multiple causes: from team members creating or updating infrastructure through the web console without backporting changes to Terraform, to unexpected actions from authenticated apps and services.
 
-Drift can have multiple causes: from developers creating or updating infrastructure through the web console without telling anyone, to uncontrolled updates on the cloud provider side. Handling infrastructure drift vs the codebase can be challenging.
+Spot discrepancies as they happen : driftctl is a free and open-source CLI that warns of infrastructure drift and fills in the missing piece in your DevSecOps toolbox.
 
-You can't efficiently improve what you don't track. We track coverage for unit tests, why not infrastructure as code coverage?
```
Comment (eliecharra) [inline README.md:35]: Why remove this sentence ? I think it's still something interesting to talk about coverage.

Comment (GeraldC13) [reply README.md:35]: okay let's keep it. Just tried to keep it short and simple, but this sentence is still valid obviously.

File: README.md [hunk 3]
```diff
@@ -28,13 +28,11 @@
 
 ## Why driftctl ?
 
-Infrastructure as code is awesome, but there are too many moving parts: codebase, state file, actual cloud state. Things tend to drift.
+Infrastructure drift is a blind spot and a source of potential security issues.
+Drift can have multiple causes: from team members creating or updating infrastructure through the web console without backporting changes to Terraform, to unexpected actions from authenticated apps and services.
 
-Drift can have multiple causes: from developers creating or updating infrastructure through the web console without telling anyone, to uncontrolled updates on the cloud provider side. Handling infrastructure drift vs the codebase can be challenging.
+Spot discrepancies as they happen : driftctl is a free and open-source CLI that warns of infrastructure drift and fills in the missing piece in your DevSecOps toolbox.
```
Comment (wbeuil) [inline README.md:34]: `as they happen:` `infrastructure drifts`

**Review by wbeuil — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -65,8 +65,10 @@ Build with ❤️️ from 🇫🇷 🇯🇵 🇬🇷 🇸🇪 🇺🇸
 
 ---
 
-## Collecting users feedback
+## Helping you get started
+
+We have 30 min slots for 1:1 [tech discussions and overviews](https://calendly.com/stephanejourdan/30min) twice a day to help you get started and collect users feedback.
 
-Have you launched driftctl ? Please take a moment to answer the [following questions](https://driftctl.typeform.com/to/fYUOOMj3) and help us improve the tool. 
+:warning: ** No sales involved **: You will deal with a technical team member of the project.
```
Comment (eliecharra) [inline README.md:72]: Remove whitespaces if you wanna a bold text `**No sales involved**`

**Review by eliecharra — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (GeraldC13) [reply README.md:72]: removing the whole sentence

File: README.md (hunk 3 above)
Comment (GeraldC13) [reply README.md:36]: good

**Review by sundowndev — APPROVED** [review]

**Review by eliecharra — APPROVED** [review]

**Review by wbeuil — APPROVED** [review]

**Pull request merged**